src/linker/configuration.py

```diff
@@ -66,13 +35,8 @@ def get_resources(self) -> Dict[str, str]:
     ####################
 
     @staticmethod
-    def _load_yaml(filepath: Path) -> Dict:
-        with open(filepath, "r") as file:
-            data = yaml.safe_load(file)
-        return data
-
     def _load_computing_environment(
-        self, computing_environment: Optional[Path]
+        computing_environment: Optional[Path],
```

Comment:
If this argument is optional, why not provide a default None value?

Reply:
An arg is always sent in (even if None from click). It seems like the difference between `Union[str, None]`, `Optional[str]`, and the more modern `str | None` is somewhat philosophical but I'll change it to Union b/c that implies to me it's not optional but None is really a valid/expected value.